harness/config/commands.yml

```diff
@@ -69,7 +69,7 @@ command('go docker vet'):
     COMPOSE_PROJECT_NAME: = @('namespace')
   exec: |
     #!bash(workspace:/)|@
-    passthru docker-compose exec -T app go vet ./...
+    passthru docker-compose exec -T app go vet -tags=benchmarks ./...
```

Comment:
```suggestion
    passthru docker-compose exec -T app go vet --tags=benchmarks ./...
```
 I think?

Reply:
both works 🤷 
`go help vet` says `The build flags supported by go vet are those that control package resolution
and execution, such as -n, -x, -v, -tags, and -toolexec.`